src/client.ts

```diff
@@ -3645,6 +3650,7 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
      * @param password - Passphrase
      * @param backupInfo - Backup metadata from `checkKeyBackup`
      * @returns key backup key
+     * @deprecated Use {@link keyFromAuthData} directly.
```

Comment:
Per my other comment: I think we just deprecate this as something we're going to stop supporting, rather than recommending an alternative.